src/components/shared/basic/ReleaseProcess/TechnicalIntegration/index.tsx

```diff
@@ -248,7 +248,7 @@ export default function TechnicalIntegration() {
             setUploadCSVError(true)
           }
         }
-        reader.readAsText(file)
+        reader.readAsText(file, 'ISO-8859-4')
```

Comment:
ISO-8859-4 is a specific encoding for north european countries (https://en.wikipedia.org/wiki/ISO/IEC_8859-4) - our files should generally be in unicode, specifically utf-8

Does `reader.readAsText(file, 'utf-8')` not work?

If it doesn't work then the file is probably in an unsupported encoding and the user should make sure to only upload files that are in supported formats.